sheduling/src/utils/api.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const host = "http://localhost:3000/login";
+const API = {
+  login: (phone, pass, success) => {
+    axios.post(host, { phone: phone, password: pass }).then(res => {
+      success(res);
```

Comment:
I would also suggest that you store the token in the local storage using something like `localStorage.setItem(res.token)`

Then on the other page you can access it from the store or the local storage as well. But since you have implemented redux persisit, is should work fine without sending it the the local storage.